railties/CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+*   `config/application.rb` now includes
+
+    ```ruby
+    config.autoload_lib(ignore: %w(assets tasks))
+    ```
+
+    In practice, this means that new 7.1 applications autoload from `lib` out of the box.
```

Comment:
Very late nitpick:

> autoload from `lib`

This wording in the docs is an indication that `autoload_from_lib` would be a better name for the method.

Reply:
That wording is not so unique. We also say "autoload the `lib` directory", or "the `app/models` directory is autoloaded by default".